features/frameworks/greentea-client/source/greentea_test_env.cpp

```diff
@@ -94,8 +94,10 @@ void _GREENTEA_SETUP_COMMON(const int timeout, const char *host_test_name, char
  *           This function is blocking.
  */
 extern "C" void GREENTEA_SETUP(const int timeout, const char *host_test_name) {
+#if defined(NO_GREENTEA)
```

Comment:
Had a thought. Shouldn't this be:

```suggestion
#if ! defined(NO_GREENTEA)
```

We want _GREENTEA_SETUP_COMMON to run if NO_GREENTEA is not set.